packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -25,19 +25,13 @@ dialog:not([open]) {
 
 dialog + .backdrop {
```

Comment:
It looks like the darker text color [pointed out in my VRT diff notes](https://github.com/CMSgov/design-system/pull/2391#issuecomment-1448977629) is coming from the `.dialog` polyfill rule. Because the polyfill is for no-longer supported browsers and for server-side rendering frameworks, maybe we should consider getting rid of the polyfill styles entirely.

Reply:
I'm down for that. Should removing it be part of this epic or a larger "clean up IE-only CSS" ticket?

Reply:
Looking deeper, the dialog polyfill is also applying its own reset to `<dialog>`, which is overriding the styles set on `<body>`. I can remove the polyfill or update it to exclude our set styles.

Reply:
Ah okay, good call